kahuna/public/stylesheets/main.css

```diff
@@ -2855,3 +2855,36 @@ FIXME: what to do with touch devices
     }
 
 }
+
+.result__select__overlay.alert {
+  background-color: #C433314D;
```

Comment:
I can see the element this styling refers to in the DOM, but it doesn't look like it has a height – so I don't see the tinted overlay:

<img width="592" alt="Screenshot 2022-11-10 at 10 56 47" src="https://user-images.githubusercontent.com/7767575/201073372-c356f2be-f77f-4286-93fb-4d61fd0e165d.png">

Having said that, I think the messaging is clear without it, and to me the overlay is problematic – it reduces the contrast of text that is already hard to read in some cases. If it's still desirable, would it be better to alter the background of the preview component? Alternatively, using [`background-blend-mode`](https://developer.mozilla.org/en-US/docs/Web/CSS/background-blend-mode) might be one way of applying a hue whilst preserving contract.

Reply:
Just noticed I have some uncommitted css. This should now be fixed